common/constants.ts

```diff
@@ -6,41 +6,94 @@ import {
   ShirtSize,
   ApplicationStatus,
   RegistrationResponse,
-} from "./types";
-
-export const EXAMPLE_GENDER = Gender.Female;
-export const EXAMPLE_EDUCATION = Education.Undergraduate;
-export const EXAMPLE_ETHNICITIES = [Ethnicity.Asian];
-export const EXAMPLE_SHIRTSIZE = ShirtSize.Small;
-export const EXAMPLE_APPLICATION_STATUS = ApplicationStatus.Declined;
```

Comment:
we should probably go through and run the formatter on all the existing files at once so these diffs don't happen – i can do this in another PR

Reply:
see #22